app/helpers/roles_helper.rb

```diff
@@ -8,4 +8,12 @@ def role_link(role)
       end
     end
   end
+
+  def delete_role_confirmation(role)
+    if role.users.any?
+      _("Warning! This will remove %{name} from %{number} users. are you sure?") % {name: role.name, number: role.users.size}
```

Comment:
Use `n_()` here for the pluralised form, i.e. "from %{number} user" (1) and "from %{number} users".